src/v1/item/typers/property/options.ts

```diff
@@ -4,10 +4,10 @@ import { RequiredOption } from '../constants/requiredOptions'
  * Common input options of all Properties
  */
 export interface PropertyOptions<
-  R extends RequiredOption = RequiredOption,
-  H extends boolean = boolean,
-  K extends boolean = boolean,
-  S extends string | undefined = string | undefined
+  Required extends RequiredOption = RequiredOption,
+  Hidden extends boolean = boolean,
+  Key extends boolean = boolean,
```

Comment:
```suggestion
  IsRequired extends RequiredOption = RequiredOption,
  IsHidden extends boolean = boolean,
  IsKey extends boolean = boolean,
```

Maybe add a `Is` prefix so it'd be more obvious that it's a boolean without looking at the implementation?

Reply:
Sure